packages/database/src/core/PersistentConnection.ts

```diff
@@ -75,6 +75,13 @@ interface OutstandingPut {
   onComplete: (a: string, b?: string) => void;
 }
 
+interface OutstandingGet {
+  action: string;
+  request: object;
+  queued?: boolean;
+  onComplete: (a: string, b?: string) => void;
```

Comment:
What are you thoughts on breaking some of the coding conventions in this repo and used descriptive names for `a` (value?)  and `b` (error?),

This might also go away if you follow my advice above.